tests/test_changeable.py

```diff
@@ -157,3 +159,96 @@ def test_custom_post_change_view(client):
     )
 
     assert b"Profile Page" in response.data
+
+
+@pytest.mark.settings(password_complexity_checker="zxcvbn")
+def test_easy_password(app, client):
+    authenticate(client)
+
+    data = (
+        '{"password": "password", '
+        '"new_password": "mattmatt2", '
+        '"new_password_confirm": "mattmatt2"}'
+    )
+    response = client.post(
+        "/change", data=data, headers={"Content-Type": "application/json"}
+    )
+    # assert response.headers["Content-Type"] == "application/json"
```

Comment:
why are the two lines commented out?

Reply:
maybe then it would be better to add some comments to explain why we have the commented code there, otherwise no-one will know it in a few months.

Reply:
Done!